jest.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
     },
     testEnvironment: 'jsdom',
     setupFiles: ['<rootDir>/jest/setup.ts', './node_modules/@react-native-google-signin/google-signin/jest/build/setup.js'],
-    setupFilesAfterEnv: ['@testing-library/jest-native/extend-expect', '<rootDir>/jest/setupAfterEnv.ts', '<rootDir>/tests/perf-test/setupAfterEnv.js'],
```

Comment:
why moving it to different file?

Reply:
@kubabutkiewicz @fabioh8010 I just followed the instructions on https://github.com/testing-library/jest-native?tab=readme-ov-file#typescript-support

Without this change, I was getting typescript errors saying that `toBeDisabled` was not part or jest